extension/js/agent/patches/patchBackboneView.js

```diff
@@ -11,49 +12,55 @@
         });
       }
     });
-  }, 200);
-
-  // @private
-  Agent.patchBackboneView = function(BackboneView) {
-    debug.log('Backbone.View detected');
+  };
 
-    Agent.patchBackboneComponent(BackboneView, function(view) { // on new instance
-      Agent.lazyWorker.push({
-        context: Agent,
-        args: [view],
-        callback: function(view) {
-          // registra il nuovo componente dell'app
-          var data = Agent.serializeView(view)
-          Agent.registerAppComponent('View', view, data);
-          Agent.sendAppComponentReport('view:change', {
-            data: data,
-            cid: view.cid
-          });
-        }
-      });
-
-      // set class name if we can
-      if (view._requirePath) {
-        view._className = _.last(view._requirePath.split('/'));
+  var onNewViewInstance = function(view) { // on new instance
+    debug.log('new view instance:', view.cid);
+    // flag to not trigger view:change when is registering
+    var isRegisteringView = true;
+    var viewChanged = _.debounce(_.partial(patchViewChanges, view), 0);
+    Agent.lazyWorker.push({
+      context: Agent,
+      args: [view],
+      callback: function(view) {
+        // registra il nuovo componente dell'app
+        var data = Agent.serializeView(view);
+        Agent.registerAppComponent('View', view, data);
```

Comment:
sendAppComponentReport` isn't needed somewhere?

Reply:
`registerAppComponent` sends a 'view:new' event that, in inspector, adds the view to the view collection tree. This already triggers the rendering, so the 'view:change' event triggered by `sendAppComponentReport` is redundant